8Knot/queries/company_query.py

```diff
@@ -42,82 +37,51 @@ def company_query(self, repos):
 
     query_string = f"""
                     SELECT
-                        c.cntrb_id,
-                        c.created_at AS created,
+                        left(c.cntrb_id::text, 15), -- first 15 characters of the uuid
+                        timezone('utc', c.created_at) AS created,
```

Comment:
This is good.

Reply:
do these projection-time modifications add significant overhead?

Reply:
++ awesome that's what I was hoping